fundamentals/fundamentals-bio/src/main/scala/izumi/functional/bio/retry/package.scala

```diff
@@ -0,0 +1,14 @@
+package izumi.functional.bio
+
+import java.time.{Instant, ZoneOffset, ZonedDateTime}
+
+package object retry {
+  type Scheduler3[F[-_, +_, +_]] = Scheduler2[F[Any, +_, +_]]
+  object Scheduler3 {
+    @inline def apply[F[-_, +_, +_]: Scheduler3]: Scheduler3[F] = implicitly
+  }
+
+  @inline def toZonedDateTime(epochMillis: Long): ZonedDateTime = {
+    ZonedDateTime.ofInstant(Instant.ofEpochMilli(epochMillis), ZoneOffset.UTC)
```

Comment:
I've moved it from `bio` package because it's kinda weird to have `Scheduler2` under `retry` and `Scheduler3` under bio package. Obviously, it would be awesome to store it with other type aliases, but
I think it makes sense to have a separate package for this one.